core/eth/contracts/Terabethia.sol

```diff
@@ -67,27 +67,26 @@ contract Terabethia is Initializable, ITerabethiaCore {
         external
         returns (bytes32)
     {
-        simpleStorage().nonce += 1;
```

Comment:
There is a PR open for this change already #27

Reply:
Yes, Its the fist commit of this branch I used it as base